index.js

```diff
@@ -0,0 +1,35 @@
+import Book from './modules/classes.js';
+
```

Comment:
- [ ] OPTIONAL: You have done a great job in this project. But I have noticed that after adding a new book, the list page is not updated, when I navigate to the list page I cannot see the new added book, I have to reload the page to see it. I recommend updating the page every time when users click on add button. You can try to use `location.reload()` method after click